gateway/dataimpl/server_v1/searchserver.go

```diff
@@ -447,9 +447,9 @@ func (s *SearchServer) translatePSQueryToCBSearchX(in *search_v1.Query) (cbsearc
 		}
 		return &cbsearchx.BooleanQuery{
 			Boost:   query.BooleanQuery.GetBoost(),
-			Must:    must,
-			MustNot: mustNot,
-			Should:  should,
+			Must:    &must,
```

Comment:
This seems like the caller should be handling the nil values rather than always sending the queries even if they don't exist?